src/web/detectors/ScaleGestureDetector.ts

```diff
@@ -145,27 +145,27 @@ export default class ScaleGestureDetector implements ScaleGestureListener {
     return true;
   }
 
-  public getCurrentSpan(): number {
-    return this.currentSpan;
-  }
+  public calculateScaleFactor(numOfPointers: number): number {
+    if (numOfPointers < 2) {
+      return 1;
+    }
 
-  public getFocusX(): number {
-    return this.focusX;
+    return this.prevSpan > 0 ? this.currentSpan / this.prevSpan : 1;
   }
```

Comment:
Since this is a trivial calculation, could we maybe change it to `public get scaleFactor`?
This wouldn't be the first calculating getter, `timeDelta` getter also performs a calculation.

Reply:
This name was changed to `calculateScaleFactor` intentionally. we do not have access to `numOfPointers` inside `ScaleGestureDetector` so we can't perform first check without having it as a parameter

Reply:
You're right, missed the parameter